.github/PULL_REQUEST_TEMPLATE/feature_or_bugfix_pr.md

```diff
@@ -0,0 +1,17 @@
+---
+name: New Feature or Bug fix
+about: Implementation of a new feature or fixes an existing one
```

Comment:
> Implementation of a new feature or fixes an existing one

This reads as `fixes an existing feature`. You probably want it more like `or fixes an issue`  instead.